installscripts/dockerfiles/jenkins/launch_jenkins_docker.sh

```diff
@@ -72,7 +72,7 @@ spin_wheel $! "Creating the Jenkins volume"
 
 # Pull the docker image from ECR
 region=`grep region ~/jazz-installer/installscripts/jazz-terraform-unix-noinstances/variables.tf  | cut -d' ' -f 9 | tr -d '"'`
-aws ecr get-login --registry-ids 108206174331 --no-include-email --region $region > ecr_login_script
+aws ecr get-login --registry-ids 108206174331 --no-include-email --region us-east-1 > ecr_login_script
```

Comment:
Since the Jenkins docker image is hosted in us-east-1 region of AWS ECR, we will need to login to us-east-1. This is relevant even when the user has selected a different region for install Jazz resources. I think we should move the image to Docker-hub soon.